From 9d9605dcc941633ad402fed2f7f8acd7fa902a58 Mon Sep 17 00:00:00 2001
From: Timple Raj M <quic_timple@quicinc.com>
Date: Tue, 30 Jan 2024 20:46:12 +0530
Subject: [PATCH] kconfig: enable PREEMPT_COUNT by default for 16M profile

enable PREEMPT_COUNT by default if IPQ_FLASH_16M_PROFILE
is true while disabling DEBUG_ATOMIC_SLEEP from qsdk.

This patch is snapshot of the commit
5c0a07baf5007a8c6435c787d6949e3d1257c887
from win.linuxopenwrt.2.1 branch and
oss/kernel/linux-ipq-5.4 project

Change-Id: Ie93feb55be8d240d94c0d70077a99a40a7d65c23
Signed-off-by: Timple Raj M <quic_timple@quicinc.com>
---
 kernel/Kconfig.preempt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/Kconfig.preempt b/kernel/Kconfig.preempt
index c2f1fd95a821..33952c7e2665 100644
--- a/kernel/Kconfig.preempt
+++ b/kernel/Kconfig.preempt
@@ -88,6 +88,7 @@ endchoice
 
 config PREEMPT_COUNT
        bool
+       default y if IPQ_FLASH_16M_PROFILE
 
 config PREEMPTION
        bool
-- 
2.34.1

